reference/Playback-Configuration-API.yaml

```diff
@@ -811,17 +848,21 @@ components:
         - $ref: "#/components/schemas/PlaybackApiKeys"
         - type: object
           properties:
+            id:
+              type: string
+              minLength: 1
+              example: ghb4ok1234
             apiKey:
               type: string
               minLength: 1
               example: apikeyvalue
-            id:
+            name:
               type: string
               minLength: 1
-              example: ghb4ok1234
+              example: some-unique-name
```

Comment:
minor. can we make it for example 'web' to make it a bit more meaningful for readers?